+++
title = "Risk alleles of genes with monoallelic expression are enriched in gain-of-function variants and depleted in loss-of-function variants for neurodevelopmental disorders"
date = 2017-03-07
draft = false

# Authors. Comma separated list, e.g. `["Bob Smith", "David Jones"]`.
authors = ["V Savova", "S Vinogradova", "D Pruss", "AA Gimelbrant", "LA Weiss"]

# Publication type.
# Legend:
# 0 = Uncategorized
# 1 = Conference paper
# 2 = Journal article
# 3 = Manuscript
# 4 = Report
# 5 = Book
# 6 = Book section
publication_types = ["2"]

# Publication name and optional abbreviated version.
publication = "Risk alleles of genes with monoallelic expression are enriched in gain-of-function variants and depleted in loss-of-function variants for neurodevelopmental disorders"

# Abstract and optional shortened version.
abstract = "Over 3000 human genes can be expressed from a single allele in one cell, and from the other allele—or both—in neighboring cells. Little is known about the consequences of this epigenetic phenomenon, monoallelic expression (MAE). We hypothesized that MAE increases expression variability, with a potential impact on human disease. Here, we use a chromatin signature to infer MAE for genes in lymphoblastoid cell lines and human fetal brain tissue. We confirm that across clones MAE status correlates with expression level, and that in human tissue data sets, MAE genes show increased expression variability. We then compare mono- and biallelic genes at three distinct scales. In the human population, we observe that genes with polymorphisms influencing expression variance are more likely to be MAE (P<1.1 × 10−6). At the trans-species level, we find gene expression differences and directional selection between humans and chimpanzees more common among MAE genes (P<0.05). Extending to human disease, we show that MAE genes are under-represented in neurodevelopmental copy number variants (CNVs) (P<2.2 × 10−10), suggesting that pathogenic variants acting via expression level are less likely to involve MAE genes. Using neuropsychiatric single-nucleotide polymorphism (SNP) and single-nucleotide variant (SNV) data, we see that genes with pathogenic expression-altering or loss-of-function variants are less likely MAE (P<7.5 × 10−11) and genes with only missense or gain-of-function variants are more likely MAE (P<1.4 × 10−6). Together, our results suggest that MAE genes tolerate a greater range of expression level than biallelic expression (BAE) genes, and this information may be useful in prediction of pathogenicity."
# Is this a selected publication? (true/false)
selected = true

# Digital Object Identifier (DOI)
doi = "10.1038/mp.2017.13"

# Does this page contain LaTeX math? (true/false)
math = false

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder. 
[image]
+++


